src/google/protobuf/util/field_comparator.h

```diff
@@ -183,6 +183,7 @@ class PROTOBUF_EXPORT DefaultFieldComparator : public FieldComparator {
   // can use ResultFromBoolean() to convert that boolean to a ComparisonResult
   // value.
   bool CompareBool(const FieldDescriptor& field, bool value_1, bool value_2) {
+    (void)field;
```

Comment:
I think a slightly more succinct way of accomplishing this is just to avoid giving the parameter a name at all, so for example the function signature could like this:

`bool CompareBool(const FieldDescriptor& /* unused */, bool value_1, bool value_2) {`

Could you try doing it that way in this file? (I know in `arenastring.h` it would not work very well because of the `ifdef`).

Reply:
done :)